applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -128,7 +145,8 @@ class TrilinosBlockBuilderAndSolver
                                   typename TLinearSolver::Pointer pNewLinearSystemSolver)
         : BuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver>(pNewLinearSystemSolver),
           mrComm(rComm),
-          mGuessRowSize(GuessRowSize)
+          mGuessRowSize(GuessRowSize),
+          mSynchronizeDofSet(false)
```

Comment:
please use the definition of the member to initialize it, otherwise it needs to be added to every CTor